.storybook/storybook-data.js

```diff
@@ -368,6 +368,19 @@ module.exports = [
 			},
 		},
 	},
+	{
+		id: 'global-notifications--user-input-success-notification',
+		kind: 'Global/Notifications',
+		name: 'User Input Success Notification',
+		story: 'User Input Success Notification',
+		parameters: {
+			fileName: './stories/notifications.stories.js',
+			options: {
+				hierarchyRootSeparator: '|',
+				hierarchySeparator: {},
+			},
+		},
+	},
```

Comment:
@kostyalmm This is not really necessary to add a Storybook story, it adds the story to the visual regression tests which would be a separate requirement. It's not a blocker that you added this here, but going forward please only add new Storybook stories to Storybook, not the VRT. We'll have to revisit which stories we include in VRT overall since there is a lot of legacy bloat there, while other important things are missing.

Just FYI 👍